conf/messages.cy

```diff
@@ -865,7 +865,8 @@ provideTraderEori.error.required=Enter the trader''s EORI number
 provideTraderEori.error.tooLong=EORI number is too long
 provideTraderEori.error.tooShort=EORI number is not long enough
 provideTraderEori.error.notGB=Enter EORI number beginning with GB
-provideTraderEori.error.general=EORI country code can only be GB followed by 12 digits, like GB123456123456
+provideTraderEori.error.specialCharacters=Enter an EORI number in the correct format, like GB123456789010
```

Comment:
Error message set earlier was correct, can we not have same here as well  - EORI country code can only be GB followed by 12 digits, like GB123456123456.